docs/user/integration/sap-cloud-logging/README.md

```diff
@@ -103,8 +103,11 @@ The Telemetry module supports the convenient shipment of applications and access
 
 ### Set Up Access Logs
 
-1. By default, Istio access logs are disabled in Kyma. To enable Istio access logs selectively for your workload, follow [Enable Istio access logs](https://kyma-project.io/#/istio/user/operation-guides/02-30-enable-istio-access-logs).
-   As a result, you can analyze access logs in the default SAP Cloud Logging dashboards shipped for SAP BTP, Kyma runtime.
+By default, Istio sidecar injection and Istio access logs are disabled in Kyma. With the following steps, you can analyze access logs of your workload in the default SAP Cloud Logging dashboards shipped for SAP BTP, Kyma runtime.
```

Comment:
```suggestion
The Istio module is added to your Kyma cluster by default, but with Istio sidecar injection and Istio access logs disabled. 

To analyze access logs of your workload in the default SAP Cloud Logging dashboards shipped for SAP BTP, Kyma runtime, you must enable them:
```

Reply:
I really think it is not important to talk about the istio module in general but only about if you enabled it for your workload. I somehow like the original better

Reply:
If we make sure that Istio module is mentioned as a prerequisites for ingesting Istio logs etc, then no need to mention it here.

Reply:
```suggestion
By default, Istio sidecar injection and Istio access logs are disabled in Kyma.

To analyze access logs of your workload in the default SAP Cloud Logging dashboards shipped for SAP BTP, Kyma runtime, you must enable them:
```